src/common/components/MovieSearchPrompt.vue

```diff
@@ -63,9 +69,14 @@ import { ref, computed, watch } from "vue";
 import { MovieSearchIndex } from "@/common/types/models";
 import { useSearch } from "@/service/useTMDB";
 
-const { defaultList, defaultListTitle } = defineProps<{
+const {
+  defaultList,
+  defaultListTitle,
+  includeSearch = true,
+} = defineProps<{
   defaultList: MovieSearchIndex[];
   defaultListTitle: string;
+  includeSearch?: boolean;
```

Comment:
I'm struggling to understand what `includeSearch` is and why it's needed

Reply:
By default, this component searches TMDB to provide search results. In the nominations page, we only want this modal to display our review movies for the year, so I added this prop to control whether there are search results from TMDB.